hack/verify-golint.sh

```diff
@@ -18,30 +18,13 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+readonly VERS="v1.33.0"
```

Comment:
nit: rename this to VERSION (or maybe something even more descriptive

Reply:
fixed